src/iconvert/iconvert.cpp

```diff
@@ -20,7 +20,7 @@
 
 
 using namespace OIIO;
-
+using OIIO::Strutil::print;
```

Comment:
Probably should add in a `#include <OpenImageIO/strutil.h>` and remove the now unnecessary `<iostream>` include.